cloudinit/user_data.py

```diff
@@ -108,10 +119,11 @@ def find_ctype(payload):
                     ctype_orig = None
                     was_compressed = True
                 except util.DecompressionError as e:
-                    LOG.warning("Failed decompressing payload from %s of"
-                                " length %s due to: %s",
-                                ctype_orig, len(payload), e)
-                    continue
```

Comment:
We still need this `continue`, I think, as we won't necessarily be raising an exception in `_handle_error`.